ci_config.json

```diff
@@ -199,6 +199,17 @@
       "windows": false
     }
   },
+  "libglvnd": {
+    "_comment": "Windows needs pthreads. macOS/LLD does not support --version-script",
```

Comment:
```suggestion
    "_comment": "upstream explicitly only supports Unix-like OSes excluding macOS",
```

Reply:
Btw that linker failure can probably be done on Linux as well by using CC_LD=lld

Reply:
macOS doesn't use lld by default though, it uses ld64, which netoriously doesn't support version scripts

I just tested on linux with gcc + lld and clang + lld and it works fine.

Reply:
Right, this is absolutely a Darwin issue not an LLD one.

lld is supposed to be fairly compatible with bfd so it would be rather awkward if it was missing support for one of the most significant features used in practice... I'm hardly surprised they support it. :P